File: BasicCourseOOP/1. CursoPOOUber/JS/Driver.js
class Driver extends Account {
    constructor(name,document,email,password){
        super(name,document,email,password)
    }
    printDataDriver(){
        console.log(`Nombre: ${this.name}`);
        console.log(`Document: ${this.document}`);
        console.log(`E-mail: ${this.email}`);
        console.log(`Password: ${this.password}`);
    }
}

File: BasicCourseOOP/1. CursoPOOUber/Java/Driver.java
public class Driver extends Account{
    public Driver(String name, String document, String email, String password){
        super(name,document,email,password);
    }

    String printDataDriver(){
        return ("Document driver: "+document+" Name drive: "+name+" E-mail: "+email+" Password: "+password);
    }
}
